common/planners/src/plan_expression_chain.rs

```diff
@@ -181,16 +193,11 @@ impl ExpressionChain {
                     "Action must be a non-aggregated function.",
                 ));
             }
-            Expression::Sort { expr, .. } => {
-                self.add_expr(expr)?;
-            }
-
-            Expression::Wildcard => {}
+            Expression::Wildcard | Expression::Sort { .. } => {}
```

Comment:
sort expr is ignored?

Reply:
because sort inner expr has processed. https://github.com/datafuselabs/databend/pull/3603/files/47840ebd49bf414f77209b478e40941591a739db#diff-caf7c3e1b3cdcec7931de05d8293eede913050ddea2788261cbaebf14dc9c11fR59